typescript/vrotest/e2e/src/runner.ts

```diff
@@ -78,7 +79,7 @@ async function executeScript(cwd: string, args: string[]): Promise<void> {
 		cwd,
 		env: process.env,
 		shell: true,
-		stdio: "inherit", 
+		stdio: "inherit",
```

Comment:
Do we need the last comma here?
```suggestion
		stdio: "inherit"
```

Reply:
It is supported in newer versions of JS/TS but I agree with you - it is not necessary, let's remove it :)